### PR TITLE
Resize tabs when mouse moves away from tab bar

### PR DESCRIPTION
**Pull request by danjordan**
Here's an implementation of the Chrome tabs functionality required in issue #129.

Functionality added...

- Each tab has it's maxWidth set to it's current width when the mouse enters the tabbarview
- Each tab has it's maxWidth set to '' when the mouse leaves the tabbarview

### CODE REVIEW

**Comment by alexandernst** [comment]
Is this going anywhere? 20 days and not even a comment from an atom dev, let alone code review.

**Comment by mnquintana** [comment]
This is looking good! Would you mind adding some kind of transition to the tabs so they don't just pop to their new size?

**Comment by danjordan** [comment]
Not sure if you got a notification when I pushed the code, so just letting you know I have added an animation when the tabs move back to max-height.

**Comment by rageandqq** [comment]
Any updates on this feature?

File: styles/tabs.less [hunk 1]
```diff
@@ -17,6 +17,7 @@
     -webkit-flex: 1;
     max-width: 175px;
     min-width: 40px;
+    transition: max-width .25s ease-in;
```
Comment (simurai) [inline styles/tabs.less]: Not too critical, but how about `ease-in-out`? Then it doesn't stop too abruptly?

**Comment by simurai** [comment]
Tried it out and UX-wise it works great, especially for even sized tabs. 👍 

tabs

**Comment by mnquintana** [comment]
This was added in #344 – thanks again for all your hard work @danjordan! 👏

**Comment by mnquintana** [comment]
This should land in Atom 1.10

**Comment by adambuczynski** [comment]
@danjordan this is the best feature to land in Atom since the first release :D

**Pull request closed without merging**